mmdet3d/structures/bbox_3d/utils.py

```diff
@@ -7,6 +7,7 @@
 from torch import Tensor
 
 from mmdet3d.utils import array_converter
+from .cam_box3d import CameraInstance3DBoxes
```

Comment:
This will cause a `circular import error`

Reply:
Thank you for all the suggestions!!! I will fix it right now.